lib/app.coffee

```diff
@@ -33,6 +33,9 @@ events = require('./events')
 plugins = require('./utils/plugins')
 update = require('./utils/update')
 
+# Assign bluebird as the global promise library
+require('any-promise/register/bluebird')
```

Comment:
I mean I guess I could always wrap `streamToPromise` in a bluebird promise but I could see that causing issues on codepaths that aren't followed often, and therefore not tested as well.